cromshell

```diff
@@ -441,7 +441,7 @@ function _turtleSpinner()
   local turtRFile=$( makeTemp )
   turtle &> ${tmpTurtFile}
   sed 's#^  ##g' ${tmpTurtFile} > ${turtFile}
-  rev ${turtFile} | sed -e 's$/$#$g' -e 's#\\#/#g' -e 's$#$\\$g' -e 's#(#)#g' -e 's#)#(#g' > ${turtRFile}
+  rev ${turtFile} | sed -e 's$/$#$g' -e 's#\\#/#g' -e 's$#$\\$g' -e 's#(#)#g' -e 's#)#(#g' -e 's#>#a#g' -e 's#<#>#g' -e 's#a#<#g' > ${turtRFile}
```

Comment:
```suggestion
  rev ${turtFile} | sed -e 's#/#SLASH#g' -e 's#\\#/#g' -e 's#SLASH#\\#g' -e 's#(#OPEN_PAREN#g' -e 's#)#(#g' -e 's#OPEN_PAREN#)#g' -e 's#>#GREATER_THAN#g' -e 's#<#>#g' -e 's#GREATER_THAN#<#g' > ${turtRFile}
```

Reply:
no need to limit ourselves to single character temporaries....and I think it makes the "code" cleaner.

Reply:
The multi-character match and replacements take longer, though.

It's also a little confusing to read the line and see that only half of the `(` characters are replaced with `OPEN_PAREN`.

I think rather than this replacement a comment above the line stating why there are multiple replacement strings (for the placeholders between `<` and `>`, for example) is a good solution.

Reply:
I doubt that the time difference maters for this use, and I find it easier to read...but, I'm not a maintainer, so I just opine....

Reply:
I've compromised by using TMP as the standard replacement for everything and adding a comment.